### ISSUE TITLE
Inquiry About YOLO Enterprise License – Pricing, Usage, and Integration Options

### ISSUE THREAD

**Issue by pornpra** [ultralytics/ultralytics#21066]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

Dear Ultralytics Team,

I’m interested in learning more about the YOLO Enterprise license and would appreciate clarification on a few key points:

1. Pricing Model: Could you provide details on the pricing structure? Is it subscription-based, one-time, or usage-based? Are there different tiers based on deployment size or feature access?

2. Usage Method: Once licensed, how do we run the YOLO model—do we use it via a Python SDK with license key integration, or is it offered through a graphical interface (e.g., drag-and-drop or no-code UX/UI)?

3. Deployment Options: Can the enterprise version be deployed on our own infrastructure (e.g., on-premise, Docker, Jetson), or does it require running through your platform?

4. Feature Differences: How does the enterprise license differ in features or performance compared to the open-source YOLOv8?

5. Trial or Demo: Is there an option to test the enterprise version before purchasing, such as a trial period or demo walkthrough?

Looking forward to your response. 😀

**Comment by UltralyticsAssistant** [comment]
👋 Hello @pornpra, thank you for your interest in Ultralytics and for your detailed questions about the YOLO Enterprise license! 🚀 This is an automated response to acknowledge your inquiry—an Ultralytics engineer will follow up soon to provide further details.

In the meantime, we recommend checking our [Docs](https://docs.ultralytics.com/) for general information on YOLO capabilities and deployment options. If you have questions about using YOLO11, YOLO12, or other versions, you may also find helpful usage examples in the [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) guides.

For licensing and enterprise options, our team will be best equipped to answer specifics, but you can also connect with the Ultralytics community for peer discussions:
- [Discord](https://discord.com/invite/ultralytics) 🎧 for real-time chat
- [Discourse](https://community.ultralytics.com/) for in-depth discussions
- [Subreddit](https://reddit.com/r/Ultralytics) for community-driven threads

## Upgrade

If you are considering technical evaluations, ensure you are running the latest `ultralytics` package and all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/):

```bash
pip install -U ultralytics
```

## Environments

YOLO can be run in these up-to-date verified environments (with dependencies including [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/), and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

Thank you again for reaching out! An Ultralytics engineer will provide further information on enterprise licensing, integration, and feature differences soon.

**Comment by glenn-jocher** [comment]
Thank you for your interest in the Ultralytics Enterprise License! I can help clarify several of your questions based on our available information:

**Pricing & Structure:** The Enterprise License uses an annual fee model with organization-wide usage rights. Specific pricing is customized based on your organization's size and use case, so you'll need to [request a quote through our license form](https://www.ultralytics.com/license) to get exact pricing details.

**Usage Method:** You'll receive access to the complete Ultralytics YOLO source code portfolio (including YOLOv5, YOLOv8, YOLO11, and future releases) along with proprietary enterprise-exclusive models. This is accessed via Python SDK with your standard development workflow, not through a separate graphical interface.

**Deployment Options:** Yes, the enterprise license allows deployment on your own infrastructure including on-premise, Docker, Jetson devices, and other environments. You have full control over where and how you deploy the models.

**Feature Differences:** Enterprise models are pre-trained on significantly larger datasets with more object classes compared to open-source models, providing a better foundation for fine-tuning. You also get access to proprietary models not available in the open-source version, plus full commercial rights without AGPL-3.0 restrictions.

**Trial/Demo:** For trial availability and detailed feature demonstrations, please mention this when you [submit the license request form](https://www.ultralytics.com/license) - our team can discuss options during the consultation process.

The license includes basic support, with options for enhanced SLA and consulting services under separate agreements. Would you like to proceed with the license request form to get more specific details tailored to your use case?
